src/MoBi.Presentation/Formatters/NotAvailableValueFormatter.cs

```diff
@@ -0,0 +1,17 @@
+using MoBi.Assets;
+using OSPSuite.Core.Domain;
+
+namespace MoBi.Presentation.Formatters
+{
+   public class NotAvailableValueFormatter : NullableWithDisplayUnitFormatter
```

Comment:
this namign is weird.It's more a. ValueAllowingNaNFormatter or something like this